### ISSUE TITLE
Fetcher un fichier avec un header accept JSON+LD retourne le fichier

### ISSUE THREAD

**Issue by srosset81** [assemblee-virtuelle/semapps#1107]
Il faudrait plutôt que ça renvoie la représentation sémantique du fichier (`semapps:File`).

Il faudrait faire une exception si le fichier est lui-même du JSON+LD ?

Un avis @simonLouvet ?

**Comment by simonLouvet** [comment]
je suis d'accord sur le principe que un `accept JSON+LD` et autre accept semantique (ttl, rdf...) devrait retourner le représentation sémantique et pas le fichier brut.
Ok également pour que si le accept correpsond à l'extention du fichier, il soit retourné directement mais je ferai une deuxieme issue pour avancer par petit pas car il va falloir établir une corespondance entre les extentions, les headers sérialisés dans les fichier et e mime-type. Il ne me semble pas que cela fasse partie de le spec W3C donc ont peut faire comme on l'entends.
